chromium_src/chrome/browser/ui/omnibox/omnibox_theme.cc

```diff
@@ -0,0 +1,92 @@
+#include "chrome/browser/ui/omnibox/omnibox_theme.h"
+
+// Rewrite original function name (see header file)
+#define GetOmniboxColor GetOmniboxColor_ChromiumImpl
+#define OmniboxTint OmniboxTint_Chromium
+#include "../../../chrome/browser/ui/omnibox/omnibox_theme.cc"
+#undef GetOmniboxColor
+#undef OmniboxTint
+
+namespace {
+
+OmniboxTint_Chromium BraveTintToChromiumTint(OmniboxTint brave_tint) {
+  switch (brave_tint) {
+    case OmniboxTint::PRIVATE:
+    case OmniboxTint::DARK:
+      return OmniboxTint_Chromium::DARK;
+    case OmniboxTint::LIGHT:
+      return OmniboxTint_Chromium::LIGHT;
+    case OmniboxTint::NATIVE:
+      return OmniboxTint_Chromium::NATIVE;
+    default:
+      return OmniboxTint_Chromium::LIGHT;
+  }
+}
+
+template <class T>
+constexpr T DarkPrivateLight(OmniboxTint tint,
```

Comment:
This template is needed? I found that only SkColor is used as template argument.